### ISSUE TITLE
Copy & paste snafu in materials code?

### ISSUE THREAD

**Issue by zellski** [KhronosGroup/COLLADA2GLTF#144]
I was idly browsing GLTFMaterial.cpp when something leaped out at me:

https://github.com/KhronosGroup/COLLADA2GLTF/blob/master/GLTF/src/GLTFMaterial.cpp#L880-L885

```C++
	if (values->emissionTexture) {
		GLTF::MaterialPBR::Texture* texture = new GLTF::MaterialPBR::Texture();
		texture->texture = values->diffuseTexture;
		material->emissiveTexture = texture;
		material->emissiveFactor = new float[3]{ 1.0, 1.0, 1.0 };
	}
```

I think `values->diffuseTexture` on the third line should probably be `values->emissiveTexture`?

**Comment by lasalvavida** [comment]
@zellski, haha, you're absolutely right. This will be fixed by #145.

**Comment by lasalvavida** [comment]
Fixed by #145.
